tests/test_paths.py

```diff
@@ -15,6 +15,8 @@
 """
 import unittest
 
+from cl_dev_tools import TestCase
```

Comment:
should remove - `cl-dev-tools` are not available here AFAIK. Should just use `unittest.TestCase`.